packages/components/README.md

```diff
@@ -283,5 +290,55 @@ yarn publish
 ```
 
 This will prompt you for the new version.  Choose a version increment in accordance with [SemVer](https://semver.org/).  This command will
-update the `package.json` file and commit that change.  Then you can do a `git push` to get the update into the remote repository. (Note,
-you could instead use `npm publish`, but you will have to update the `package.json` file manually before using that command.)
+update the `package.json` file and commit that change.  Then you can do a `git push` to get the update into the remote repository.
+
+## Merging changes into master
+
+1. Message the Frontend dev room chat about starting the pull request merge. This is to make sure two people aren't
+merging at the same time which might result in conflicting package version numbers.
+1. Do one final merge of the `master` branch into your feature branch for `labkey-ui-components`.
+1. Run one final lint of your changes, `yarn run lint-branch-fix`, and review the changes applied.
+1. Update the `releaseNotes/labkey/components.md` file with what will be your release version number and release date.
+1. Run the commands to build, test, and publish: `yarn build`, `yarn test`, `yarn publish`.
+1. Push your final set of commits from `labkey-ui-components` to github so that TeamCity can do a final run of the jest tests.
+1. Update any LabKey module `package.json` files where you are using / applying these changes with this final release version
+number (then do the regular `npm install` for that module, build, etc. and push those `package.json` and `package-lock.json` file
+changes to github as well).
+1. Remove any of the alpha package versions from [Artifactory](https://artifactory.labkey.com/artifactory/webapp/#/home)
+that you had published during development for this feature branch.
```

Comment:
Might want to add a sub-bullet or two here on how to do this removal.

Reply:
done